src/ShallowTraversal.js

```diff
@@ -108,34 +107,34 @@ export function nodeMatchesObjectProps(node, props) {
   return isSubset(propsOfNode(node), props);
 }
 
+function buildSelectorPredicate(selector) {
+  const nodes = getAst(selector).nodes[0];
+  return AND(nodes.map(node => {
+    switch (node.type) {
+      case 'class':
```

Comment:
The strings come from `postcss-selector-parser`, and aren't exported. But I could definitely put a map in util if you think that's best!